draft-ietf-add-split-horizon-authority.xml

```diff
@@ -476,59 +476,55 @@
               a signed response containing "ns1.example.com" and
               "ns2.example.com". The client then performs full DNSSEC
               validation locally.</li>
-            <li><strong>Step 3</strong>: As the DNSSEC validation is successful and
-              the network-provided nameservers are the same as the names in
-              the DNSSEC response, and the network-designated resolver's
-              certificate includes at least one of the names returned in the
-              DNSSEC response, the client has finished validation that the
-              nameservers signaled in <xref target="I-D.ietf-add-dnr"/>
-              and <xref target="RFC8801"/> are owned and managed by the
-              same entity that published the NS records on the Internet. The
-              endpoint will then use that information from <xref
-              target="I-D.ietf-add-dnr"/> and <xref
-              target="RFC8801"/> to resolve names within dnsZones.</li>
+            <li><strong>Step 3</strong>: The DNSSEC validation is successful and
+              the network-provided DNS servers are listed in the signed NS
+              record for <tt>example.com</tt>, so these ADNs are authorized.
+              When the client connects using an encrypted transport as indicated
+              in <xref target="I-D.ietf-add-dnr">DNR</xref>, it will authenticate
+              the server to its name using TLS, and send queries to resolve any
```

Comment:
I suggest replacing TLS with PKIX or refer to https://datatracker.ietf.org/doc/html/rfc8310#section-8.

Reply:
OK, added reference.